### ISSUE TITLE
Update CONTRIBUTING.md

### ISSUE THREAD

**Issue by Kefaku** [JustWhatsTheData/justwhatsthedata.github.io#5]
Currently, the top of CONTRIBUTING.md still uses information relevent to justdeleteme, which is not relevant to this project, e.g.:

>  url: The url of the account-deletion page. If no such page exists, the url should be a contact or help page explaining the process of account deletion.

or:

> domains: This is used by the [Chrome extension](https://github.com/jdm-contrib/justdelete.me-chrome-extension)

This should be updated to match justwhatsthedata instead.

**Comment by coreystone** [comment]
Agreed, the project seems abandoned for now unfortunately. With better guidelines it could be great, but the premise is pretty ambitious and I imagine it would be hard to compare services to each other.
